packages/uicore/src/components/Filters/components/FilterSingleSelectDropDown/FiltersSelectDropDown.css

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2024 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+.main {
+  .dropdownButton {
+    border-radius: var(--spacing-2);
+    padding: 0px var(--spacing-4);
+    height: var(--spacing-8);
+    transition: all 200ms ease-out;
+    transition-property: border-color, background-color;
+    &.minWidth {
+      column-gap: var(--spacing-small);
+    }
+    &.withBorder {
+      border: 1px solid var(--grey-200);
+      &.selected {
+        border-color: var(--grey-200);
+      }
+      .labelWrapper {
+        .label {
+          font-size: var(--font-size-small);
+          font-weight: 500;
+          color: var(--grey-800);
+          outline: none;
+        }
+      }
+    }
+    .labelWrapper {
+      flex-shrink: 1;
+      column-gap: var(--spacing-3);
+      :global(.bp3-popover-wrapper) {
+        display: inline-flex;
+      }
+      .label {
+        font-size: var(--font-size-normal);
+        font-weight: 400;
+        color: var(--grey-900);
+      }
+      .counter {
+        background-color: var(--grey-100);
+        padding: 0 var(--spacing-xsmall);
+        border-radius: var(--spacing-xsmall);
+        font-size: 13px;
+      }
+    }
+  }
+}
+.popover {
```

Comment:
The css was picked from existing Multiselect component. There may be scope to reduce the css, this will be optimised later